t/ui/16-activity-view.t

```diff
@@ -84,13 +148,49 @@ subtest 'Current jobs' => sub {
     like $driver->get_title(), qr/Activity View/, 'search shown' or return;
     my $results = $driver->find_element_by_id('results');
     my @entries = $results->children('.list-group-item');
-    is scalar @entries, 5, '5 jobs' or return diag explain $results->get_text;
+    is scalar @entries, 19, '19 jobs' or return diag explain $results->get_text;
+
+    my @rows = $driver->find_elements('#results .list-group-item');
+    for my $row (@rows) {
+        my @links = $row->children('a');
+        my $href  = $links[0]->get_attribute('href');
+        unless ($href =~ m{/tests/(\d+)}) {
+            fail("Link '$href' is not a test url");
+            next;
+        }
+        my $id        = $1;
+        my $exp       = delete $state_result{$id};
+        my $exp_class = $exp->[3];
+        my @i         = $row->children('i');
+        my @class     = split ' ', $i[0]->get_attribute('class');
+        ok((grep { $_ eq $exp_class } @class), "classname contains $exp_class")
+          or diag(Data::Dumper->Dump([\@class], ['class']));
+    }
+    if (keys %state_result) {
+        fail "Missed the following jobs in the list:";
```

Comment:
This block should likely be marked as uncoverable.

Reply:
done